source/_static/scss/includes/_aside.scss

```diff
@@ -30,7 +30,7 @@
     }
 }
 
-.sidebar-active {
+div.sidebar-active {
```

Comment:
`sidebar-active` active class is set on the body tag to toggle the sidebar. Currently the toggling isn't working due to element `div` ref. Can we keep the selector simple as `.sidebar-active`?